internal/api/appreciation.go

```diff
@@ -9,62 +9,68 @@ import (
 	"github.com/joshsoftware/peerly-backend/internal/app/appreciation"
 	"github.com/joshsoftware/peerly-backend/internal/pkg/apperrors"
 	"github.com/joshsoftware/peerly-backend/internal/pkg/dto"
+	log "github.com/joshsoftware/peerly-backend/internal/pkg/logger"
 	"github.com/joshsoftware/peerly-backend/internal/pkg/utils"
-	logger "github.com/sirupsen/logrus"
 )
 
 func createAppreciationHandler(appreciationSvc appreciation.Service) http.HandlerFunc {
 	return http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
 		var appreciation dto.Appreciation
 		err := json.NewDecoder(req.Body).Decode(&appreciation)
 		if err != nil {
-			logger.Errorf("Error while decoding request data : %v", err)
+			log.Errorf(req.Context(),"Error while decoding request data : %v", err)
 			err = apperrors.JSONParsingErrorReq
 			dto.ErrorRepsonse(rw, err)
 			return
 		}
 
+		log.Info(req.Context(),"createAppreciationHandler: request: ",req)
 		err = appreciation.ValidateCreateAppreciation()
 		if err != nil {
-			logger.Errorf("Error while validating request data : %v", err)
+			log.Errorf(req.Context(),"Error while validating request data : %v", err)
 			dto.ErrorRepsonse(rw, err)
 			return
 		}
 
 		resp, err := appreciationSvc.CreateAppreciation(req.Context(), appreciation)
 		if err != nil {
-			logger.Errorf("err : %v", err)
+			log.Errorf(req.Context(),"createAppreciationHandler: err : %v", err)
 			dto.ErrorRepsonse(rw, err)
 			return
 		}
+		log.Debug(req.Context(),"createAppreciationHandler: response: ",resp)
+		log.Info(req.Context(),"Appreciation created successfully")
 		dto.SuccessRepsonse(rw, http.StatusCreated, "Appreciation created successfully", resp)
 	})
 }
 
 func getAppreciationByIDHandler(appreciationSvc appreciation.Service) http.HandlerFunc {
 	return http.HandlerFunc(func(rw http.ResponseWriter, req *http.Request) {
-
 		vars := mux.Vars(req)
 		apprID, err := strconv.Atoi(vars["id"])
 		if err != nil {
+			log.Errorf(req.Context(),"Error while decoding appreciation id : %v", err)
 			dto.ErrorRepsonse(rw, err)
 			return
 		}
-
+		
+		log.Info(req.Context(),"getAppreciationByIDHandler: request: ",req)
 		resp, err := appreciationSvc.GetAppreciationById(req.Context(), int32(apprID))
 		if err != nil {
-			logger.Errorf("err : %v", err)
+			log.Errorf(req.Context(),"getAppreciationByIDHandler: err : %v", err)
 			dto.ErrorRepsonse(rw, err)
 			return
 		}
+
+		log.Debug(req.Context(),"getAppreciationByIDHandler: request: ",resp)
```

Comment:
fix the typo error and check all logs for the same.

Reply:
resolved